boot/zephyr/prj.conf

```diff
@@ -11,4 +11,3 @@ CONFIG_MBEDTLS_CFG_FILE="config-boot.h"
 CONFIG_HEAP_MEM_POOL_SIZE=16384
 
 CONFIG_FLASH=y
-CONFIG_SOC_FLASH_STM32F4=y
```

Comment:
That's probably because of https://github.com/Linaro/zephyr/commit/942edd44751ce45dd4b979302b648e5d90b68a1b. I'm hoping the upstream version to be merged soon, then we will have the right fix at the right place.